__DEFINES/__compile_options.dm

```diff
@@ -44,6 +44,8 @@
 //#define MAP_OVERRIDE 15
 // wheelstation.dm
 //#define MAP_OVERRIDE 16
+// tgstation-sec.dm
```

Comment:
This seems unintended.

Reply:
Forgot to mention it, there is now a MAP_OVERRIDE for the purpose of testing because for some reason it wasn't there.